source/stylesheets/components/_sponsors.scss

```diff
@@ -19,6 +19,8 @@
 .sponsors img {
   border: none;
   margin-bottom: 1em;
+  width: 100%;
+  height: auto;
```

Comment:
Currently they look like:

![image](https://user-images.githubusercontent.com/325384/93860410-fac9d480-fcb6-11ea-81f9-aac7606dc7b1.png)

I'm guessing that element has changed at some point, but setting CSS like above overrides the explicit HTML attributes to make them fit a bit more nicely.

I'm 50/50 on if this was relevant to this PR, but as I was touching that file already it took my focus.

The end result is it looks more like:

![image](https://user-images.githubusercontent.com/325384/93860825-893e5600-fcb7-11ea-92d7-ff68b21011e9.png)

Plus also allows for when the proportion of the images is different.

Reply:
I think it's fine in this PR and if it fixes something that's cool.  I guess the new grid layout is ironically less flexible than the old "float a div" - suspect the old div just expanded to what it contained, or we'd fixed it at a given size.  When adding sponsor logos for the sidebar I make sure they're all the same width (give or take a few px probably), but the ratios are all over because of that.

Thanks for explaining though!  Looks good.